app/mailers/welcome_mailer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class WelcomeMailer < ActionMailer::Base
+  default from: 'scholar@uc.edu'
+  def welcome_email(email)
+    new_user = User.find_by_email(email)
```

Comment:
@13miciunasl Instead of passing resource.email from after_sign_up_path_for, just pass resource (the entire user object).  That way you won't need to do a find_by_email here.  You also won't need the .nil? check if you're not doing a find.